actions/user/deleteSocialData.js

```diff
@@ -0,0 +1,4 @@
+export default function deleteSocialData(context, payload, done) {
```

Comment:
You could add more than one method to an action (or file) by omitting the "default" and exporting several functions. Reimport is handled through `import {abc, xyz} from ...`